homeassistant/components/sensor/melissa.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support for Melissa climate Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.melissa/
+"""
+import logging
+
+from homeassistant.components.melissa import DOMAIN, DATA_MELISSA
+from homeassistant.const import TEMP_CELSIUS, STATE_UNKNOWN
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = [DOMAIN]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the melissa sensor platform."""
+    sensors = []
+    connection = hass.data[DATA_MELISSA]
+    devices = connection.fetch_devices().values()
+    for device in devices:
+        sensors += [
+            MelissaTemperatureSensor(
+                device,
+                connection
+            ),
+            MelissaHumiditySensor(
+                device,
+                connection
+            )
+        ]
```

Comment:
Is it intended here to pass an array of arrays to add_devices?

Reply:
It does not, its just a flat list.
```
>>> devices = []
>>> for i in range(5):
...     devices += [i]
...
>>> print(devices)
[0, 1, 2, 3, 4]
```